### PR TITLE
Copy file to pictures folder

### PR DESCRIPTION
**Pull request by colincarter**
This PR adds a windows only function `copyToPictures` to support [this ](https://github.com/axsy-dev/react-app/issues/9138)feature in react-app.

`KnownFolders.PicturesLibrary.Path` is empty.  [This ](https://stackoverflow.com/questions/38874259/windows-storage-knownfolders-musiclibrary-path-returns-empty)post, which talks about the music library, but seems to apply to all media libraries, says they are virtual and should be accessed through the StorageFile object.

(also includes prettier changes)

### CODE REVIEW

File: windows/RNFS-vnext/RNFSManager.cs [hunk 1]
```diff
@@ -379,6 +379,21 @@ public async void copyFile(string filepath, string destPath, JSValue options, IR
             }
         }
 
+        [ReactMethod]
+        public async void copyToPictures(string filepath, IReactPromise<bool> promise)
+        {
+            try
+            {
+                var srcFile = await StorageFile.GetFileFromPathAsync(filepath);
```
Comment (colincarter) [inline windows/RNFS-vnext/RNFSManager.cs:387]: Need to use `StorageFile` to copy to Pictures dir as `KnowFolders.PicturesLibrary.Path` returns null.

File: index.d.ts [hunk 2]
```diff
@@ -244,51 +241,55 @@ export function copyAssetsFileIOS(
...
-export const TemporaryDirectoryPath: string
-export const LibraryDirectoryPath: string
-export const PicturesDirectoryPath: string
-export const FileProtectionKeys: string
+  filepath: string,
+  mtime?: Date,
+  ctime?: Date
+): Promise<void>;
+
+export function copyToPictures(path: string): Promise<bool>;
```
Comment (colincarter) [inline index.d.ts:283]: Change added here

File: FS.common.js [hunk 3]
```diff
@@ -616,6 +771,14 @@ var RNFS = {
     return RNFSManager.scanFile(path);
   },
 
+  copyToPictures(path: string): Promise<boolean> {
```
Comment (colincarter) [inline FS.common.js:774]: Change added here

File: package.json
Comment (rtoddsullivan) [inline package.json:1]: Just checking, does this need yarn.lock?

**Review by rtoddsullivan — APPROVED** [review]

**Pull request merged**